### ISSUE TITLE
include-what-you-use results

### ISSUE THREAD

**Issue by huwpascoe** [citra-emu/citra#2984]
Raw output from building with include-what-you-use.
https://gist.github.com/huwpascoe/c2da0a487164702ad1ee7ba1d5a70629

**Comment by mailwl** [comment]
call for wwylele. Is cryptopp bulletproof for security?

**Comment by wwylele** [comment]
@mailwl no, and I already mentioned when introducing it that its internal module management was a mess, which doesn't have much to do with "security" in term of crypto. Even if it does break security, we don't care much about because we are not, and will not likely use it for actual secure purpose.
